src/constants/memory/wram.asm

```diff
@@ -3556,28 +3567,28 @@ wFile3DeathCountHigh::
 wFile3DeathCountLow::
   ds 1 ; DC05
 
-; Amount of health for file 1, needs to be wFile1MaxHealth * 8 or the game crashes while drawing the file selection menu
+; Amount of health for file 1, needs to be wFile1MaxHearts * 8 or the game crashes while drawing the file selection menu
 wFile1Health::
   ds 1 ; DC06
 
-; Amount of health for file 2, needs to be wFile2MaxHealth * 8 or the game crashes while drawing the file selection menu
+; Amount of health for file 2, needs to be wFile2MaxHearts * 8 or the game crashes while drawing the file selection menu
 wFile2Health::
   ds 1 ; DC07
 
-; Amount of health for file 3, needs to be wFile3MaxHealth * 8 or the game crashes while drawing the file selection menu
+; Amount of health for file 3, needs to be wFile3MaxHearts * 8 or the game crashes while drawing the file selection menu
 wFile3Health::
   ds 1 ; DC08
 
 ; Amount of maximum health for file 1, used to draw the hearts on the file selection menu
```

Comment:
```suggestion
; Amount of maximum hearts for file 1, used to draw the hearts on the file selection menu
```

?

Reply:
Quite right, I'll fix it for all three file comments and rebase